src/MOLECULE/atom_vec_template.cpp

```diff
@@ -29,20 +29,21 @@ AtomVecTemplate::AtomVecTemplate(LAMMPS *lmp) : AtomVec(lmp)
   atom->molecule_flag = 1;
   atom->molindex_flag = 1;
   atom->molatom_flag = 1;
+  atom->q_flag = 1;
```

Comment:
This change must be reverted.

Atom style template is equivalent to atom style molecular. 
The equivalent of atom style full is achieved through using atom style hybrid.

Reply:
done